novawallet/Modules/Swaps/Setup/SwapSetupPresenter.swift

```diff
@@ -25,6 +25,15 @@ final class SwapSetupPresenter: SwapBasePresenter {
 
     private var feeIdentifier: SwapSetupFeeIdentifier?
     private var slippage: BigRational
+    private var issues: [SwapSetupViewIssue] = [] {
+        didSet {
+            provideDetailsViewModel()
+        }
+    }
+
+    private var detailsAvailable: Bool {
+        !issues.contains(.noLiqudity) && quoteArgs != nil
```

Comment:
Better to check that quoteResult is not error. In this case we also don't need save issues locally